src/fetch_package.js

```diff
@@ -68,6 +68,18 @@ export function flushCache() {
     }
 }
 
+/**
+ * Flush the cache of anything that hasn't been used in the last 15 minutes.
+ */
+export function flushUnusedCache() {
+    /**
+     * Guard this in case we never enabled caching.
+     */
+    if (args.useCache) {
+        cache.flushUnused(15 * 60);
```

Comment:
SUPER NIT:  Might be nice to have a comment closer to the code (or a constant with a meaningful name) stating how long we want this to be, that way folks looking at it can compare what the code says with what the comments says was intended.